packages/experiments/src/components/fluent/FluentDepths.ts

```diff
@@ -0,0 +1,7 @@
+export namespace Depths {
+  export const depth0 = '0 0 0 0 transparent';
```

Comment:
Not sure if they exist right now, but it'd great to add a comment with usage/examples for each of these when we have them.

Reply:
We'll have that documented on the website. Hadn't considered adding comments in the code. That could potentially be useful, I'd just worry about maintaining it in two places.

Reply:
That's fair, but I think it'd be really useful for Intellisense. I hear you on documenting it in two places, but the code really should have usage information. Once we have them checked in it hopefully shouldn't change much.